src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -124,13 +124,15 @@ def find_issues(self, path):
         '''
         Run mozlint through mach, without gecko-env
         '''
+        full_path = os.path.join(self.repo_dir, path)
 
         # Run mozlint on a file
         command = [
+            'gecko-env',
             './mach', 'lint',
             '-f', 'json',
             '--quiet',
-            path
+            full_path
```

Comment:
is this needed if the working idrectory is repo_dir?

Reply:
It's an attempt to help mozlint have a unique index per file, maybe they 'll fix the issue at some point

Reply:
Have you filed a bug on bugzilla about this?

Nit: if the attempt didn't work, go back to the old way to reduce the changes in the PR.